sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -25,7 +29,10 @@ public CouchConfig getCouchConfig(String db) {
                         " is required to run tests with Cloudant.");
             }
             return config;
-        } else {
+        } else if (SPECIFIED_COUCH) {
+            return SpecifiedCouch.defaultConfig(db);
```

Comment:
it follows the pattern we've used previously with cloudant, it has its trade offs, there are a lot more config options, a url would be easier, but may be confusing is you don't know how to pass a username and password through the url